tests/override_test_sqlalchemy_store.py

```diff
@@ -13,6 +13,10 @@ def test_log_param_max_length_value(store: SqlAlchemyStore, monkeypatch):
     ()
 
 
+def test_set_tag(store: SqlAlchemyStore, monkeypatch):
+    ()
+
+
```

Comment:
@dsuhinin I just tried implementing this, and it seems to require a lot of test-specific dependencies from the MLflow side. For example, the `_run_factory` function makes use of another helper function `_create_experiments` and so on, which would clutter our tests if we ported them over. And they are private so I don't think we have a choice other than to implement them directly on our `override_test_sqlalchemy_store.py`. I feel that implementing these would clutter that file too much, maybe we could have another file for test utils? 🤔 At any rate, I feel that it would be a pretty large change so we could make a separate issue/PR to sort it out!